dds_web/api/user.py

```diff
@@ -775,6 +786,36 @@ def delete_user(user):
             db.session.rollback()
             raise ddserr.DatabaseError(message=str(err))
 
+    @staticmethod
```

Comment:
Will this function be used by other classes in the future?

Reply:
Yeah that's what I was thinking